### DISCUSSION TITLE
How to contribute to the etherpad modlue?

### DISCUSSION THREAD

**Discussion by JuTiger-Lee** [ether/etherpad-lite#5082] [Q&A] [answered]
I created a module called ep_line_spacing. I want to contribute this to ehterpad but I don't know how.

**Comment by rhansen** [comment, marked as answer]
You have already contributed it. :slightly_smiling_face:   Simply publishing to npm is sufficient.

**Reply by JuTiger-Lee** [reply]
Thank you. We will do our best to contribute more.
